www/app/Http/Controllers/Api/ConversationController.php

```
@@ -297,6 +297,36 @@ public function stream(Request $request, Conversation $conversation): JsonRespon
+                return response()->json([
+                    'success' => false,
+                    'error' => '/compact requires an active Claude Code session. Send a message first to start one.',
+                ], 422);
+            }
+            $jobOptions['is_compact_command'] = true;
+        }
+
+        // Initialize stream state BEFORE cleanup to prevent race condition
+        // This ensures clients won't see 'not_found' after cleanup and before job starts
+        RequestFlowLogger::log('controller.stream.initializing', 'Initializing stream state in Redis');
+        $this->streamManager->startStream($conversation->uuid, [
```

PHPStan (2.1.46)

[error] Access to an undefined property App\Models\Conversation::$uuid.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)